data/Dockerfiles/acme/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.6
+
+LABEL maintainer "Andre Peters <andre.peters@servercow.de>"
+
+RUN apk add --update --no-cache \
+      acme-client \
+      docker \
```

Comment:
Yes. docker sucks up 90MB of 100MB in the disk image. I'm aware of that problem. But I just have not known yet another api. Let's have a look. Thanks.